### PR TITLE
Update to newer atoum versions

### PR DESCRIPTION
**Pull request by jubianchi**
atoum 2.8 allows us to remove the bootstrap when it's only used to
require the autoloader (see atoum/atoum#605)

atoum 3.0 allows us to run tests on latest PHP versions.

I also had to add doctrine/common as a dependency to make the tests run.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,17 +15,12 @@
...
-    "repositories": [
-        {
-            "type": "composer",
-            "url": "http://satis.m6web.fr"
-        }
-    ],
     "require": {
-        "friendsofsymfony/rest-bundle"   : "~1.1"
+        "friendsofsymfony/rest-bundle"   : "~1.1",
+        "doctrine/common": "^2.7"
```
Comment (Oliboy50) [inline composer.json]: I think you should require `"doctrine/annotations": ">= 1.0"` instead

**Comment by jubianchi** [comment]
Should be good now ;)

File: composer.json [hunk 2]
```diff
@@ -15,17 +15,12 @@
...
-        }
-    ],
     "require": {
-        "friendsofsymfony/rest-bundle"   : "~1.1"
+        "friendsofsymfony/rest-bundle"   : "~1.1",
+        "doctrine/common": "@stable"
     },
     "require-dev": {
-        "atoum/atoum"       : "@stable",
+        "atoum/atoum"       : "^2.8|^3.0",
```
Comment (Oliboy50) [inline composer.json]: should be `||` (double pipe for OR), but it's fine

Comment (jubianchi) [inline composer.json]: fixed

**Comment by omansour** [comment]
v2.2.0

**Pull request merged**